src/Network.cpp

```diff
@@ -862,9 +862,41 @@ Network::Netresult Network::get_scored_moves(
     if (ensemble == DIRECT) {
         assert(rotation >= 0 && rotation <= 7);
         result = get_scored_moves_internal(planes, rotation);
+    } else if (ensemble == MULTI_AVG && rotation > 1) {
+        // MULTI_AVG rotation is the # of random rotations to provide
+        int num_rotations = rotation;
+
+        assert(num_rotations >= 1 && num_rotations <= 8);
+        std::vector<int> r_list{0, 1, 2, 3, 4, 5, 6, 7};
+        // Need to account for extra 2 in m_squaresize.
+        std::vector<float> policy((BOARD_SIZE + 2) * (BOARD_SIZE + 1));
```

Comment:
I don't think a separate `policy` is needed to accumulate as you could just use the one from the first rotation that you store as `result` (so use `result.policy`) ?

Reply:
Fixed!